### PR TITLE
Bulk workers kv endpoint bindings

### PR DESCRIPTION
**Pull request by Michael9127**
Adds bindings for new bulk workers KV endpoint: https://api.cloudflare.com/#workers-kv-namespace-write-multiple-key-value-pairs

Check err return of rate limiting

Testing:
* Manually tested against personal CF namespace
* Added unit test to cover new method

### CODE REVIEW

**Comment by ags799** [comment]
can you add me as reviewer @Michael9127

**Comment by Michael9127** [comment]
@ags799 I am not sure how to do that, i dont know if i have the required permissions. No edit button near listed reviewers for me.

File: workers_kv.go [hunk 1]
```diff
@@ -141,6 +141,26 @@ func (api *API) WriteWorkersKV(ctx context.Context, namespaceID, key string, val
 	return result, err
 }
 
+// WriteWorkersKVBulk writes multiple KVs at once.
+//
+// API reference: https://api.cloudflare.com/#workers-kv-namespace-write-multiple-key-value-pairs
+func (api *API) WriteWorkersKVBulk(ctx context.Context, namespaceID string, kvs []byte) (Response, error) {
```
Comment (ags799) [inline workers_kv.go]: maybe `kvs` should be `*WriteWorkersKVBulkRequest`:
```
type WriteWorkersKVBulkRequest = []struct{
  Key string `json:"key"`
  Value string `json:"value"`
  Expiration int `json:"expiration,omitempty"`
  ExpirationTTL int `json:"expiration_ttl,omitempty"`
}
```

File: workers_kv_test.go [hunk 2]
```diff
@@ -190,6 +191,41 @@ func TestWorkersKV_WriteWorkersKV(t *testing.T) {
...
+	}`
+
+	mux.HandleFunc(fmt.Sprintf("/accounts/foo/storage/kv/namespaces/%s/bulk", namespace), func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "PUT", r.Method, "Expected method 'PUT', got %s", r.Method)
+		w.Header().Set("content-type", "application/json")
+		fmt.Fprintf(w, response)
+	})
+
+	serializedKVs, err := json.Marshal(kvs)
+	assert.Nil(t, err)
```
Comment (ags799) [inline workers_kv_test.go]: probably want to `require.NoError`

File: workers_kv_test.go [hunk 3]
```diff
@@ -190,6 +191,41 @@ func TestWorkersKV_WriteWorkersKV(t *testing.T) {
...
+	want := successResponse
+	res, err := client.WriteWorkersKVBulk(context.Background(), namespace, serializedKVs)
+
+	if assert.NoError(t, err) {
+		assert.Equal(t, want, res)
+	}
```
Comment (ags799) [inline workers_kv_test.go]: you can
```
require.NoError(t, err)
require.Equal(t, want, res)
```

**Comment by a-robinson** [comment]
Should we add Workers KV to [the list of supported features on the main README](https://github.com/cloudflare/cloudflare-go#features)?

**Comment by Electroid** [comment]
Alternative solution propose here: #310. Tried to decouple the expiry and bulk changes, but it ended up being easier to put them together.

**Comment by Michael9127** [comment]
> Should we add Workers KV to [the list of supported features on the main README](https://github.com/cloudflare/cloudflare-go#features)?

Done

**Comment by Michael9127** [comment]
Since #310 is still in limbo, can this be merged in the meanwhile?

**Comment by daltoniam** [comment]
Anyway we can get this reviewed again and merged? It is a dependency for one of our projects and we would rather use the mainline library instead of a fork.

**Comment by patryk** [comment]
Thanks for the contribution and I appreciate patience.

**Pull request merged**